src/Service/SMSSender.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Service;
+
+use App\Model\Message;
+
+class SMSSender implements SenderInterface
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function supports(Message $message)
+    {
+        return $message->type == Message::TYPE_EMAIL;
+    }
+
+    /**
+     * {@inheritDoc}
+     *
+     * @param Message $message
```

Comment:
This should be moved to the interface method documentation if `@inheritDoc` is used
```suggestion
```